src/odemis/gui/cont/tabs/cryo_chamber_tab.py

```diff
@@ -220,6 +232,22 @@ def __init__(self, name, button, panel, main_frame, main_data):
                                                                      self.panel.ctrl_sample_target_tmp,
                                                                      events=wx.EVT_COMMAND_ENTER)
 
+    def _enable_advanced_movement_btns(self):
+        """Enable the advanced movement buttons based on the posture manager flag"""
+        if self._role != 'meteor':
+            return
+
+        adv_movement_enabled = self.posture_manager._flag_advanced_movement
```

Comment:
If you use `_flag_advanced_movement` outside of the PostureManager, then don't mark it private.